tests/PhoneValidatorTest.php

```diff
@@ -345,10 +346,11 @@ public function testValidatePhoneLenient()
         ]));
     }
 
+    /**
+     * @expectedException \Propaganistas\LaravelPhone\Exceptions\InvalidParameterException
+     */
     public function testValidatePhoneFaultyParameters()
     {
-        $this->setExpectedException('Propaganistas\LaravelPhone\Exceptions\InvalidParameterException');
```

Comment:
Please change into this the `setExpectedException()` method and update composer.json accordingly to require phpunit `^5.2`